Omnibus.md

```diff
@@ -626,7 +626,7 @@ is not yet destroyed]{.new}. Some spells, such as
 [Assassin's Blade](#assassins-blade), may accelerate the act of [destroying a
 body]{.new}.
 
-After the [body is destroyed]{.new}, [the character whose body has been
+After the [body is destroyed the character whose body has been
```

Comment:
The comma should've stayed between `destroyed` and `the`.

Reply:
Agreed. Removing the comma wasn't part of my suggested fix to this line.

Reply:
Woops.